x-pack/plugins/infra/public/pages/metrics/hosts/components/controls_content.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { ControlGroupInput } from '@kbn/controls-plugin/public';
+import { ViewMode } from '@kbn/embeddable-plugin/public';
+import { Filter, TimeRange } from '@kbn/es-query';
+import { LazyControlsRenderer } from './lazy_controls_renderer';
+import { useControlPanels } from '../hooks/use_control_panels_url_state';
+
+interface Props {
+  timeRange: TimeRange;
+  dataViewId: string;
+  filters: Filter[];
+  query: {
+    language: string;
+    query: string;
+  };
+  setPanelFilters: React.Dispatch<React.SetStateAction<null | Filter[]>>;
+}
+
+export const ControlsContent: React.FC<Props> = ({
+  timeRange,
+  dataViewId,
+  query,
+  filters,
+  setPanelFilters,
+}) => {
+  const { setControlPanels, controlPanel } = useControlPanels(dataViewId);
+
+  return (
+    <LazyControlsRenderer
+      getCreationOptions={async ({ addDataControlFromField }) => {
```

Comment:
@elastic/kibana-presentation I saw that the `input` parameter was replaced with `getCreationOptions` in this [PR](https://github.com/elastic/kibana/pull/146189), really nice progress there 🙌 
I tried to use `addDataControlFromField` to add the panels like in your example but in my case, I want to prefill the control group selections from a URL parameter (Currently I am persisting the whole panels object in the URL state to achieve that) Do you know if I can archive something similar with the latest Controls API changes or it's not currently available?